src/Stratis.Bitcoin.Features.Consensus/CoinviewPrefetchManager.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+using Stratis.Bitcoin.Features.Consensus.CoinViews;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus
+{
+    /// <summary>Fetches coins for blocks that are likely to be validated in the near future to speedup full validation.</summary>
+    public class CoinviewPrefetchManager : IDisposable
```

Comment:
I prefer the name to be `CoinviewPrefetcher` manager is used for in another context

Reply:
fixed